DDG4/python/DDSim/Helper/Physics.py

```diff
@@ -11,7 +11,7 @@ def __init__( self ):
     super(Physics, self).__init__()
     self._rangecut = 0.7*mm
     self.list ="FTFP_BERT"
```

Comment:
you could also fix list -> _list if you are already fixing this

Reply:
Ok, done